Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst

```diff
@@ -105,6 +105,11 @@ the mutual authentication feature:
 
    mutual-authentication-example
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation and demo of Mutual Authentication in Cilium, check out `eCHO episode 100: Next-gen mutual authentication in Cilium  <https://www.youtube.com/watch?v=60epwCxO8G4&list=PLDg_GiBbAx-mY3VFLPbLHcxo6wUjejAOC&index=80&t=2024s>`__.
```

Comment:
Seems to be the wrong link (Gateway API)

Reply:
Ah yes! good catch, will fix.